tools/wasp-cli/waspcmd/cmd.go

```diff
@@ -60,7 +60,11 @@ func DefaultWaspNodeFallback(node string) string {
 }
 
 func getDefaultWaspNode() string {
-	waspSettings := viper.Sub("wasp").AllSettings()
+	waspConfigSubTree := viper.Sub("wasp")
+	if waspConfigSubTree == nil {
+		log.Fatalf("no wasp-cli config file at default location")
+	}
+	waspSettings := waspConfigSubTree.AllSettings()
```

Comment:
The error message can be misleading, since you can have a `wasp-cli.json` with a missing `wasp` section.

```suggestion
	waspSection := viper.Sub("wasp")
	waspSettings := map[string]interface{}{}
	if waspSection != nil {
		waspSettings = waspSection.AllSettings()
	}
```